binding/java/test/ai/picovoice/porcupine/PorcupineTest.java

```diff
@@ -90,18 +142,30 @@ void testProcess() throws IOException, UnsupportedAudioFileException, PorcupineE
 
                 ByteBuffer.wrap(pcm).order(ByteOrder.LITTLE_ENDIAN).asShortBuffer().get(porcupineFrame);
                 int result = porcupine.process(porcupineFrame);
-                assertTrue(result == -1 || result == 0);
+                assertTrue(result >= -1);
                 if (result >= 0) {
                     results.add(result);
                 }
             }
         }
 
-        assertTrue(results.size() == 1 && results.get(0) == 0);
+        assertEquals(results, expectedResults);        
     }
 
     @Test
-    void testProcessMultiple() throws IOException, UnsupportedAudioFileException, PorcupineException {
+    void testSingleKeyword() throws IOException, UnsupportedAudioFileException, PorcupineException {
+        porcupine = new Porcupine.Builder()
+                .setAccessKey(accessKey)
+                .setBuiltInKeyword(Porcupine.BuiltInKeyword.PORCUPINE)
+                .build();
+
+        runProcess(
+            "porcupine.wav",
+            new ArrayList<>(Arrays.asList(0)));
```

Comment:
I would also delete the instance of porcupine at the end of each test (since that's a test in itself):
```java
porcupine.delete();
```

Reply:
there is a @AfterEach for that I think.

Reply:
Ah, I see! I hadn't seen it in the diff, so I didn't remember it was there. Yes, that'll take care of the delete then.